symfony/framework-bundle/3.3/bootstrap.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require __DIR__.'/vendor/autoload.php';
+
+if (!isset($_SERVER['APP_ENV'])) {
+    $_SERVER['APP_ENV'] = isset($_ENV['APP_ENV']) ? $_ENV['APP_ENV'] : null;
+}
+
+if ('prod' !== $_SERVER['APP_ENV']) {
+    if (!class_exists(Dotenv::class)) {
+        throw new RuntimeException('The "APP_ENV" environment variable is set not to "prod". Please run "composer require symfony/dotenv" to load the ".env" files configuring the application.');
```

Comment:
```diff
- The "APP_ENV" environment variable is set not to "prod"
+ The "APP_ENV" environment variable is not set to "prod"

Reply:
Previously, it was: `'The "APP_ENV" environment variable is not defined. You need to set it or run "composer require symfony/dotenv" to load it from a ".env" file.'`

Now, we're always requiring the Dotenv component in non prod env. Is that expected?

Reply:
fixed thanks. about non-prod envs: yes to me, that's conscious change in previous PR.